srcpkgs/apparmor/template

```diff
@@ -34,7 +34,7 @@ pre_build() {
 	cp ${FILESDIR}/profiles/* profiles/apparmor.d/
 
 	# use the correct syslog path
-	vsed -i utils/logprof.conf -e 's,logfiles = .*,logfiles = /var/log/socklog/kernel/current,'
+	vsed -i utils/logprof.conf -e 's,logfiles = .*,logfiles = /var/log/audit/audit.log /var/log/socklog/kernel/current /var/log/syslog /var/log/messages,'
```

Comment:
Please split the expression into its own line.

Reply:
moved to the logprof patch